src/token/history/history.resolver.ts

```diff
@@ -0,0 +1,58 @@
+import { Args, Int, Mutation, Resolver } from '@nestjs/graphql'
+import { Prisma } from '@prisma/client'
+import { Roles } from '../../auth/decorators/roles.decorator'
+import { TokenHistory } from '../../token/history/entities/token-history.entity'
+import { HistoryTokenService } from '../../token/history/history.service'
+
+@Resolver(() => TokenHistory)
+export class TokenResolver {
```

Comment:
You should to Add `getAllTokenHistories` with pagination ability. (For example, watch News module)

Reply:
i did it